packetbeat/protos/sip/plugin.go

```diff
@@ -160,7 +159,6 @@ func (p *plugin) buildEvent(m *message, pkt *protos.Packet) (*beat.Event, error)
 	fields["status"] = status
 
 	var sipFields ProtocolFields
-	sipFields.Timestamp = time.Now().UnixNano()
```

Comment:
Just wondering--do we set `event.created`? Because this looks like it was basically trying to do that.

Reply:
I don't believe there is any event.created in Packetbeat, but it does have an `event.start` and `event.end` which should be pretty close (except maybe for PCAPs).